client.go

```diff
@@ -251,7 +251,13 @@ func (a apiClient) Send(ctx context.Context, e any) (string, error) {
 	return res[0], nil
 }
 
-func (a apiClient) SendMany(ctx context.Context, e []any) ([]string, error) {
+func (a apiClient) SendMany(ctx context.Context, e []any) (ids []string, err error) {
+	go func() {
```

Comment:
should this be a defer? I was learning about how `recover` works while reading this PR and it looks like it can only recover from the goroutine it's executing within?